openid_connect_configuration.conf

```diff
@@ -0,0 +1,60 @@
+# OpenID Connect configuration
```

Comment:
The file itself is not included from frontend.conf nor openid_connect.server_conf.